File: modules/expo-alarms/src/ExpoAlarmsModule.ts
import { NativeModule, requireNativeModule } from 'expo';

import type { ExpoAlarmsModuleEvents } from './ExpoAlarms.types';

declare class ExpoAlarmsModule extends NativeModule<ExpoAlarmsModuleEvents> {
  getTheme: () => string;
}

// This call loads the native module object from the JSI.
export default requireNativeModule<ExpoAlarmsModule>('ExpoAlarms');

File: modules/expo-alarms/android/src/main/java/expo/modules/alarms/ExpoAlarmsModule.kt
package expo.modules.alarms

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class ExpoAlarmsModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("ExpoAlarms")

    Function("getTheme") {
      return@Function "system"
    }
  }
}
